ftw/usermigration/tests/test_migration_form.py

```diff
@@ -116,6 +138,45 @@ def test_can_use_mapping_source_adapters(self, browser):
         self.assertEquals(1, len(self.uf.searchUsers(id='new_john')))
 
     @browsing
+    def test_can_use_pre_and_post_migration_hooks(self, browser):
+        gsm = getGlobalSiteManager()
+        gsm.registerAdapter(
```

Comment:
I recommend to use the [component registry isolation layer](https://github.com/4teamwork/ftw.testing#component-registry-isolation-layer) when registering adapters in the test. It stacks the registry for each test and removes the components on test tear down (default Plone setup doesn't).